files.sh

```diff
@@ -2,7 +2,7 @@
 
 fn nash_complete_paths(parts, line, pos) {
 	partsz   <= len($parts)
-	last     <= -expr $partsz - 1
+	last     <= -echo $partsz - 1 | bc
```

Comment:
Forget it, it was already fixed in the `multiple-returns` branch, we really have to merge that 😅